docker/ckan/Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN virtualenv $CKAN_HOME && \
     chown -R www-data:www-data /var/www && \
     git clone --depth 1 --branch $BRANCH https://github.com/ckan/ckan /project/ckan && \
     git clone --branch stable --depth 1 https://github.com/ckan/datapusher /project/datapusher && \
+    $CKAN_HOME/bin/pip install -e git+https://github.com/deniszgonjanin/ckan-service-provider#egg=ckanserviceprovider && \
```

Comment:
@JackMc this is the last thing holding up this PR, would you confirm that `install --clean` leaves us in a broken state? And if so, other than forcing install of this branch like we do for ckanapi is there another way we could fix this problem?